api/api/models/audio.py

```diff
@@ -368,6 +368,7 @@ class AudioDecisionThrough(AbstractMediaDecisionThrough):
         to_field="identifier",
         on_delete=models.CASCADE,
```

Comment:
I think this should be changed to do nothing, so that while the media object is gone, we still keep a record of the identifiers.
```suggestion
        on_delete=models.DO_NOTHING,
```